.github/workflows/ci.yml

```diff
@@ -123,6 +123,88 @@ jobs:
       - run: make man
         working-directory: src/github.com/containerd/containerd
 
+  # Make sure binaries compile with other platforms
+  crossbuild:
+    name: Crossbuild Binaries
+    needs: [project, linters, protos, man]
+    runs-on: ubuntu-18.04
+    timeout-minutes: 10
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - goos: linux
+            goarch: arm64
+          - goos: linux
+            goarch: arm
+            goarm: "7"
+          - goos: linux
+            goarch: arm
+            goarm: "5"
+          - goos: freebsd
+            goarch: amd64
+          - goos: freebsd
+            goarch: arm64
+          - goos: windows
+            goarch: arm
+            goarm: "7"
```

Comment:
Who uses Windows on 32-bit ARM? 
Also, can we have Windows arm64?

Reply:
Our iot edge team does.
Go is not supported for windows/arm64.